lib/axon/losses.ex

```diff
@@ -1024,4 +1024,170 @@ defmodule Axon.Losses do
       end
     )
   end
+
+  @doc """
+  WIP
+  y_pred should be log_softmax output of shape (B, T, D),
+  y_true should be of shape (B, S),
+  where
+  B - batch size,
+  T - sequence length,
+  D - dictionary size, including blank
+  S - maximum target length
```

Comment:
To be consistent with other functions in this module, this should go under an `Argument Shapes` section

Reply:
I was planning to rewrite it after review. Description is WIP.